### ISSUE TITLE
[6.5.x] WSL: URL doesn't open in the browser

### ISSUE THREAD

**Issue by Semigradsky** [storybookjs/storybook#18086]
**Describe the bug**
URL doesn't open in the browser.
There is an error in the console:
```
ERR! Could not open http://localhost:6005/ inside a browser. If you're running this command inside a
ERR! docker container or on a CI, you need to pass the '--ci' flag to prevent opening a
ERR! browser by default.
```

**To Reproduce**
Run `start-storybook -p 6005` on WSL. (https://docs.microsoft.com/en-us/windows/wsl/)

**System**
```
Environment Info:

  System:
    OS: Linux 5.10 Ubuntu 20.04.3 LTS (Focal Fossa)
    CPU: (8) x64 Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
  Binaries:
    Node: 16.14.0 - ~/.nvm/versions/node/v16.14.0/bin/node
    Yarn: 3.2.0 - /usr/bin/yarn
    npm: 8.3.1 - ~/.nvm/versions/node/v16.14.0/bin/npm
``` 

**Additional context**
Reproduced for v6.5.0-beta-0 and v6.5.0-beta-1. All works fine for v6.4.22

**Comment by valentinpalkovic** [comment]
Which WSL version do you use? WSL 1 or WSL 2? Because I am using WSL 2 and I cannot reproduce this issue locally. The browser is opening without any problems.

**Comment by Semigradsky** [comment]
I have WSL 2. Still reproduced for me on v6.5.0-beta.4.

I can try to debug if someone can advise what the package here used for opening in the browser.

**Comment by Semigradsky** [comment]
Fixed in #18305
